WordPress/src/main/res/values/dimens.xml

```diff
@@ -170,7 +170,7 @@
     <dimen name="reader_simple_post_image_width">72dp</dimen>
 
     <dimen name="reader_filter_bottom_sheet_handle_height">15dp</dimen>
-    <dimen name="reader_remove_filter_icon_width">32dp</dimen>
+    <dimen name="reader_remove_filter_icon_width">48dp</dimen>
```

Comment:
What do you think about removing this dimension altogether and utilizing `min_touch_target_sz` since it's the same?

Reply:
Agree 👍! Changed [here](1e031da3cbdd5063fb448f7f8cdf0da86e412271)